src/components/floating-emojis/FloatingEmoji.js

```diff
@@ -24,73 +27,83 @@ const FloatingEmoji = ({
   top,
   wiggleFactor,
 }) => {
-  const { opacity, rotate, scale, translateX, translateY } = useMemoOne(() => {
-    const animation = timing({ duration, easing: EasingNode.elastic() });
-    const progress = interpolate(animation, {
-      inputRange: [0, 1],
-      outputRange: [0, distance],
-    });
+  const animation = useSharedValue(0);
+
+  animation.value = useMemoOne(
```

Comment:
I think it should be in useLayoutEffect since the animation is started even before the actual render.